ci-cd/github-actions-with-vercel-cicd/.github/workflows/e2e-release-production.yaml

```diff
@@ -0,0 +1,17 @@
+name: E2E-Release-Production
+on: [deployment_status]
+jobs:
+  e2e:
+    if: github.event.deployment.environment == 'Production' && github.event_name == 'deployment_status' && github.event.deployment_status.state == 'success'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Placeholder for running e2e tests via Github Actions (Cypress, Playwright, etc)
+        run: sleep 30
+  release:
+    needs: e2e
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Vercel CLI
+        run: npm install --global vercel@latest
+      - name: Alias Production URL
+        run: vercel alias --token=${{ secrets.VERCEL_TOKEN }} set $VERCEL_URL https://github-actions-with-vercel-cicd-production.vercel.app/
```

Comment:
Correct. The domain should be set in the Custom Domains only. If it's set in the project's domains then it will be assigned a branch it's subsequently deployed via Vercel's CI/CD and break this flow.

Reply:
ok yeah that what I was about. Can we add this to README please

Reply:
I've updated the readme with an additional note about not assigning in the Project's Domain and only the Custom Domains with a link.